repose-aggregator/src/docs/asciidoc/style-guide.adoc

```diff
@@ -1,3 +1,55 @@
 = Style Guide
```

Comment:
The proper noun **Repose** should be bolded everywhere.

Prefer block titles to explanation text.